test/specs/reference/tests/05.reference_values.js

```diff
@@ -50,9 +52,159 @@ exports.execute = function (options) {
+                                '<p><img src="http://example.com/4005.png" alt="image"></p>\n',
+                                '<p><img src="/4005.png" alt="image with size" width="400" height="400"></p>\n',
+                                '<p><a href="" download="">download link</a></p>\n',
+                                '<figure class="embed-block" style=""><figcaption class="embed-caption" style="">&lt;a href=‘javascript:alert();’&gt;&lt;/a&gt; caption</figcaption><iframe src="http://example.com/iframe" width="300" ></iframe></figure>',
+                                '<p><strong>This is some markdown</strong> with some <code>code</code> and a <a href="http://www.example.com">link</a></p>\n',
+                                '<p><strong>Name is :</strong> &lt;a href=\'javascript:alert();\'&gt;&lt;/a&gt;<br>\n<strong>This is some markdown</strong> with some <code>code</code> and a <a href="http://www.example.com">link</a></p>\n',
+                                '',
+                                '<p><a href="http://example.com/Senior">Senior</a></p>\n',
+                                '',
+                                '',
+                                expectedMomentValue
+                                ],
+                "isHTML" : [false, true, true, true, true, true, true, true, true, true, true, true, true, true]
+                },
+            "test7": {
+                "rowValue" : ["id=4006, name= <script>alert();</script>, some_gene_sequence= GATCGATCGCGTATT, some_invisible_column= Sophomore"],
+                "expectedValue" : [
+                                '4006',
+                                '<h2>&lt;script&gt;alert();&lt;/script&gt;</h2>\n',
+                                '<p><a href="/%3Cscript%3Ealert();%3C/script%3E">link</a></p>\n',
+                                '<p><img src="http://example.com/4006.png" alt="image"></p>\n',
+                                '<p><img src="/4006.png" alt="image with size" width="400" height="400"></p>\n',
+                                '<p><a href="" download="">download link</a></p>\n',
+                                '<figure class="embed-block" style=""><figcaption class="embed-caption" style="">&lt;script&gt;alert();&lt;/script&gt; caption</figcaption><iframe src="http://example.com/iframe" width="300" ></iframe></figure>',
+                                '<p><strong>This is some markdown</strong> with some <code>code</code> and a <a href="http://www.example.com">link</a></p>\n',
+                                '<p><strong>Name is :</strong> &lt;script&gt;alert();&lt;/script&gt;<br>\n<strong>This is some markdown</strong> with some <code>code</code> and a <a href="http://www.example.com">link</a></p>\n',
+                                '<code>GATCGATCGC GTATT</code>',
+                                '<p><a href="http://example.com/Sophomore">Sophomore</a></p>\n',
+                                '',
+                                '',
```

Comment:
This line for example. You can add a comment after it that suggests the actual value will be attached later.